app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="<%= dom_id food %>" class="d-flex justify-content-between foods">
+  <p>
+    <%= link_to food.name, user_food_path(current_user, food) %>
+  </p>
+
+  <p>
+    <%= food.quantity %>
+  </p>
+
+  <p>
+    <%= food.measurement_unit %>
+  </p>
+
+  <p>
+    $<%= food.price %>
+  </p>
+  <%= button_to "delete", user_foods_path(food), method: :delete %>
```

Comment:
- [ ] Good job adding the delete functionality as suggested by the previous reviewer 👍🏽 . The only challenge left is that the button is not deleting the food from the list. KIndly check it out

![image](https://user-images.githubusercontent.com/63473584/219502735-32d52dd1-6443-4091-96a5-4d2c878fa22b.png)

Reply:
i did that thank you for the review